### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by over-geek**
## TODO-List: List Structure

In this milestone, I:
- Set up project with webpack
- Created index.html with page markup
- Added function to index.js file to render tasks dynamically on page when page is loaded or reloaded
- Loaded CSS by CSS loader and added style to page

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  },
+];
+
+const addTask = (task) => {
+  list.innerHTML += `
+    <li class="task">
+      <div class="task-info">
+        <input type="checkbox">
+        <p>${task.description}</p>
+      </div>
```
Comment (AdnanOlarmmi) [inline src/index.js:31]: - [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  {
+    index: 1,
+    description: 'Complete To Do list project',
+    completed: false,
+  },
+];
+
+const addTask = (task) => {
+  list.innerHTML += `
+    <li class="task">
```
Comment (AdnanOlarmmi) [inline src/index.js:27]: - [ ] Kindly add the index to the list items also, this will ensure the lists is arranged numerically 😃.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @over-geek, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- List items are rendered dynamically.
- Webpack configured appropriately.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @over-geek       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Good implementation of the required changes👍 
- Descriptive readme👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**